src/main/scala/diplomaticobjectmodel/model/OMPorts.scala

```diff
@@ -41,17 +41,17 @@ trait AMBA extends OMProtocol
 
 case class AXI4(
   specification: Option[OMSpecification],
-  _types: Seq[String] = Seq("AXI4", "AMBA",  "OMProtocol")
+  _types: Seq[String] = Seq("AXI", "AMBA",  "OMProtocol")
```

Comment:
Are you sure you want to make this change? I'm certain this is going to break a lot of things that are expecting to match based on that `4` being present in the name.

Reply:
agree with @richardxia

Reply:
fair enough - reverting back